lib/defaults/defaults.go

```diff
@@ -252,6 +252,15 @@ const (
 	// is accessible
 	ClusterCheckTimeout = 5 * time.Second
 
+	// ClusterOperatorTimeout specifies the timeout for a ClusterOperator.
+	ClusterOperatorTimeout = 10 * time.Second
+
+	// StatusCollectionTimeout specifies the timeout for collecting gravity status.
+	StatusCollectionTimeout = 20 * time.Second
```

Comment:
I think it's a bit on the side of optimism. This value does not take any context into consideration (network latency etc.) and should be more accommodating in my opinion.

Reply:
Wasn't sure what values to use for the default timeouts. I've increased the status and audit collection timeout to one minute. What do you think would be a safe value for timeouts?